NTupleAnalyzer_nanoAOD.py

```diff
@@ -174,7 +174,7 @@
 if nonisoswitch==True or emuswitch==True or quicktestswitch==True:
 	print 'NOT performing systematics...'
 	_variations = ['']  # For quicker tests
-# _variations = ['']  # For quicker tests
+_variations = ['']  # For quicker tests
```

Comment:
I'm not sure how this got in here but I don't like it. If you want to run with no systematics you just put the string QuickTest in the name of the process you create. Please delete this line completely, then we are ready to merge.